uwblueprint/abtc
# frontend: create navbar component

## Notion ticket link
[Navbar](https://www.notion.so/uwblueprintexecs/Create-Responsive-NavBar-41e94dc2ae8d4310a7e77df26ccb21ce?pvs=4)



## Steps to test
1. add navbar component to page and build (i.e. add `<NavBarAdmin/>` or `<NavBarVolunteer/>`)
2. ensure links are working and navbar is responsive



## What should reviewers focus on?
* need second opinion on styling. do we have an official desgin for this? i just based it off of wireframes for now..

## Checklist
